### ISSUE TITLE
set not working with template literal value

### ISSUE THREAD

**Issue by rajasegar** [bigskysoftware/_hyperscript#50]
Using template literal values for set fails and the browser hangs 
```
<input type="text" id="txtName"/>
<button  type="button" _="on click set message to `hello ${#txtName.value}` then call alert(message)">Say Hello</button>
```

**Comment by benpate** [comment]
This also crashes for me.  But, I was able to make it work (mostly) if I switch the back-tick (`` ` ``) for a single quote (`'`).  [Here's a code-pen that "works"](https://codepen.io/benpate/pen/yLVxdNJ).  The one exception is that the space between the word "hello" and the inserted variable is lost.  I haven't yet dug deeper to see why.

I don't know if hyperscript *is supposed to* work with back-ticks or not, but it's definitely breaking.

@1cg -- I have two suggestions on this issue:

1) there seems like some kind of infinite loop error happening when hyperscript tries to parse the original string with back-ticks.  At a minimum, this should throw an error instead of crashing.  Same goes for the missing space that seems like it's getting optimized away by the string interpolation.

2) I think it would work better to follow Javascript on this one:  Strings delimited with `"` and `'` would be used as-is, with no variable substitution.  Strings delimited with `` ` `` would use some form of string interpolation, probably just `${variable}`.  

If you agree, I'm happy to take a shot at making a PR.

**Comment by 1cg** [comment]
Yep, I fixed the whitespace bug in dev (embarassing, but it is still a 0.0.5 :)

I agree, expectations are that ` will mean template and " and ' will mean string.  I initially felt like that was a distinction w/o a difference, but it will make the parsing easier.

Ben, you'll need to modify the tokenizer and the parser a bit.  Hit me up on discord to discuss.

**Comment by 1cg** [comment]
I have fixed this in dev.
